src/index.html

```diff
@@ -7,6 +7,460 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="Moyo logo"
+        class="logo__image"
+        >
```

Comment:
Fix code style everywhere
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo logo"
          class="logo__image"
        >
```